app/admin/order.rb

```diff
@@ -0,0 +1,43 @@
+ActiveAdmin.register Order do
+  permit_params order: [:order_status]
+
```

Comment:
we need subtabs here 'in progress', 'delivered', 'cancelled'

Reply:
https://docs.google.com/document/d/15XkEVplUUUme1iraMLji28OUgZmvzduXJj3sku4u8Q4/edit